lib/generator.rb

```diff
@@ -4,13 +4,14 @@
 require 'json'
 require 'ostruct'
 
-require_relative 'exercise_testcases'
-require_relative 'exercise_testcase'
+require_relative 'exercise_test_cases'
+require_relative 'exercise_test_case'
```

Comment:
I just changed it to follow the ruby class naming convention (the class was TestCase) - if its test_case the class should be TestCase if its testcase the class should be Testcase